### PR TITLE
Update grpc to version 1.6.0

### PR DESCRIPTION
**Pull request by KindDragon**

### CODE REVIEW

File: ports/protobuf/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: protobuf
-Version: 3.3.0-3
+Version: 3.3.0-4
```
Comment (codicodi) [inline ports/protobuf/CONTROL]: It looks like the actual protobuf version is 3.4.0
https://github.com/KindDragon/vcpkg/blob/8afb92408b9fc9a98cdaca9e4407145f07a74b21/ports/protobuf/portfile.cmake#L3

Comment (KindDragon) [reply ports/protobuf/CONTROL]: Thanks

**Comment by alexkaratarakis** [comment]
Thanks for the updates!

**Comment by Seelge** [comment]
`x64-windows` works but `x64-windows-static` fails for me.

Additionally, a mixed triplet with
`set(VCPKG_TARGET_ARCHITECTURE x64)`
`set(VCPKG_CRT_LINKAGE dynamic)`
`set(VCPKG_LIBRARY_LINKAGE static)`
fails too, although this should be identical to `x64-windows` since lib linkage is set to static in the portfile.

**Pull request merged**